src/sandstorm/control-socket.capnp

```diff
@@ -0,0 +1,75 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2020 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0xd4c5feffbd79e908;
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+using Util = import "util.capnp";
+
+interface Controller {
+  # Bootstrap interface offered by sandstorm on the control socket.
+  # This is used by some commands to control a running sandstorm
+  # server.
+  #
+  # This interface is not currently implemented. For now the commands
+  # referenced below use the `devmode` socket, and speak an ad-hoc
+  # protocol that provides similar functionality.
+  #
+  # The old protocol exists for historical reasons: when the devmode
+  # socket was first implemented, capnproto did not support passing
+  # file descriptors with a message. Now it does, so there is no
+  # need for a custom solution, and moving to capnproto will improve
+  # extensibility and maintainability.
+
+  const socketPath :Text = "/var/sandstorm/socket/control";
+  # Path to the control socket, relative to the sandstorm install (normaly
+  # /opt/sandstorm).
+
+  devShell @0 () -> (shellFds :List(FileDescriptor), cancel :Util.Handle);
```

Comment:
I think handles used in this way are usually called `handle` elsewhere in Sandstorm protocols, rather than `cancel`. Not sure if it's worth being consistent.

Reply:
Done.